theme/login/resources/js/script.js

```diff
@@ -29,12 +29,14 @@ window.onload = function () {
     if (beLogin) {
         beLogin.style.visibility = 'visible';
         const input = document.querySelector('input');
-        if(input){
+        if (input) {
             input.focus();
-  	    input.select();
+            input.select();
         }
     }
 
+    document.querySelector('#kc-social-providers h4').innerHTML = `For initial login, please use the social login of your choice. Doing so, will automatically create a user account at VSHN. For more details see <a href="https://kb.vshn.net/social-login">https://kb.vshn.net/social-login</a>.`;
```

Comment:
How hard would it be, to generically look for any URLs and replace them with a link? I dislike that the text has to be maintained at two locations.